iis-workflows/iis-workflows-citationmatching-direct/src/main/resources/eu/dnetlib/iis/workflows/citationmatching/direct/oozie_app/lib/scripts/transformer/transformer.pig

```diff
@@ -29,9 +28,8 @@ pmid_to_oaid_with_nulls = foreach documentMetadata generate externalIdentifiers#
 pmid_to_oaid_nondedup = filter pmid_to_oaid_with_nulls by originalId is not null;
 pmid_to_oaid_nondedup_groupped = group pmid_to_oaid_nondedup by originalId;
 pmid_to_oaid = foreach pmid_to_oaid_nondedup_groupped {
-     idsWithPublicationType = foreach pmid_to_oaid_nondedup generate originalId, newId, publicationTypeName;
+     idsWithPublicationType = foreach pmid_to_oaid_nondedup generate newId, publicationTypeName;
      dedupIdsWithPublicationType = DeduplicateIdsWithDocumentType(idsWithPublicationType);
---	 FIXME it works, but what if null was returned?!    
```

Comment:
I've removed all `FIXME` entries because I have already verified nulls are properly handled by PIG in this context.